### DISCUSSION TITLE
Project Pythia WIP talk

### DISCUSSION THREAD

**Discussion by jukent** [ProjectPythia/projectpythia.github.io#47] [General]
Perhaps some of you saw Sheri's request for March or April CISL WIP talks (It'll be the first Wednesday of whichever month at 1 MT). Should someone from Project Pythia discuss our work? @kmpaul @clyne

**Comment by clyne** [comment]
This is a good idea, Julia. I signed up for May; I just gave a pythia talk at the CISL town hall in December. Hopefully we'll have more to update folks on by May :-)
